packages/shell-api/src/helpers.ts

```diff
@@ -346,7 +354,7 @@ export async function getPrintableShardStatus(
     })(),
     (async (): Promise<void> => {
       // Actionlog and version checking only works on 2.7 and greater
-      let versionHasActionlog = false;
+      let versionHasActionlog = !version.currentVersion; // means that we are in a version where it is deprecated (SERVER-68888)
```

Comment:
We exclude `currentVersion` in the projection above, so `version.currentVersion` should always be undefined, no? What am I mising?

Reply:
I think the latest commit fixes it.

It keeps the currentVersion if it's on the document, until the end of the `getPrintableShardInfo` function, where we delete it, so it's not printed in the output.